client/src/frontend/components/collecting/reading-group/edit/CollectionEditor/SortableCollectables/Collectable.js

```diff
@@ -44,24 +43,46 @@ function Collectable({ responses, type, id, index, onRemove, onMove }) {
               aria-labelledby={groupLabelId}
             >
               <Remove id={id} type={type} onRemove={onRemove} />
-              <Drag
-                dragHandleProps={provided.dragHandleProps}
-                onFocus={() => setDragHandleFocused(true)}
-                onBlur={() => setDragHandleFocused(false)}
-              />
-              <div className="group-collection-editor__keyboard-actions">
-                <Move
-                  onClick={() => onMove({ id, type, direction: "up" })}
-                  onFocus={() => setKeyboardActionFocused(true)}
-                  onBlur={() => setKeyboardActionFocused(false)}
-                  direction="up"
-                />
-                <Move
-                  onClick={() => onMove({ id, type, direction: "down" })}
-                  onFocus={() => setKeyboardActionFocused(true)}
-                  onBlur={() => setKeyboardActionFocused(false)}
-                  direction="down"
+              <div
+                role="none"
```

Comment:
This isn't really necessary, since divs have no inherent role. Unless I'm missing something with the intent here?